src/schema/application/configuration/schema-configuration.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+
+// phpcs:disable Yoast.NamingConventions.NamespaceName.TooLong
+namespace Yoast\WP\SEO\Schema\Application\Configuration;
+
+use Easy_Digital_Downloads;
+use SeriouslySimplePodcasting\Integrations\Yoast\Schema\PodcastEpisode;
+use TEC\Events\Integrations\Plugins\WordPress_SEO\Events_Schema;
+use WP_Recipe_Maker;
+use WPSEO_Addon_Manager;
+use Yoast\WP\SEO\Helpers\Product_Helper;
+use Yoast\WP\SEO\Helpers\Woocommerce_Helper;
+
+/**
+ * Responsible for the schema configuration.
+ */
+class Schema_Configuration {
+
+	/**
+	 * The WooCommerce helper.
+	 *
+	 * @var Woocommerce_Helper
+	 */
+	private $woocommerce_helper;
+
+	/**
+	 * The product helper.
+	 *
+	 * @var Product_Helper
+	 */
+	private $product_helper;
+
+	/**
+	 * Schema_Configuration constructor.
+	 *
+	 * @param Woocommerce_Helper $woocommerce_helper The WooCommerce helper.
+	 * @param Product_Helper     $product_helper     The product helper.
+	 */
+	public function __construct( Woocommerce_Helper $woocommerce_helper, Product_Helper $product_helper ) {
+		$this->woocommerce_helper = $woocommerce_helper;
+		$this->product_helper     = $product_helper;
+	}
+
+	/**
+	 * Returns the schema configuration.
+	 *
+	 * @return array<string, bool|array<string, array<string, bool|string>>>
+	 */
+	public function get_configuration(): array {
+		return [
+			'isSchemaDisabledProgrammatically' => $this->is_schema_disabled_programmatically(),
+			'schemaApiIntegrations'            => $this->get_schema_api_integrations(),
+		];
+	}
+
+	/**
+	 * Gets the schema API integrations status.
+	 *
+	 * @return array<string, array<string, bool|string>> The schema API integrations status.
+	 */
+	private function get_schema_api_integrations(): array {
```

Comment:
Everything in this function is duplicated code from `Yoast\WP\SEO\Integrations\Admin::Integrations_Page` basically, let's have a single source of truth.

Best to do is:
* Take what the `Integrations_Page` class has and move it to the Schema_Configuration class (probably what you have already done here, but let's make sure that it's exactly the same data)
* Make a dependency in the integrations page for the schema configuration class and use the latter's method there too
* That way, we have the single source of truth (the method of the schema config class) that's being used both in the integrations page class and the settings integrations class